integration-tests/pom.xml

```diff
@@ -163,6 +168,12 @@
             <ansible.username>${env.ANSIBLE_USERNAME}</ansible.username>
             <ansible.password>${env.ANSIBLE_PASSWORD}</ansible.password>
             <ansible.ssl.verification.disabled>true</ansible.ssl.verification.disabled>
+
+            <!-- AWS properties -->
+            <aws.queue-name-or-arn>${env.AWS_QUEUE_NAME_OR_ARN}</aws.queue-name-or-arn>
+            <aws.region>${env.AWS_REGION}</aws.region>
+            <aws.access-key>${env.AWS_ACCESS_KEY}</aws.access-key>
+            <aws.secret-key>${env.AWS_SECRET_KEY}</aws.secret-key>
```

Comment:
what values should I use for local testing, are they captured somewhere?

Reply:
well, I've just modified a little bit the PR, though these values are still needed for the TNB framework to connect to the different resources.

There is an example in the README.md file, along with an [example file](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/46eff411988cdefb125407de46ae68c555349806/integration-tests/localconfig-example.yaml#L14-L18)

Basically, all you need to do is to provide this file with all your AWS details which will be used by the TNB framework to connect to.
Once you have that file, all you need to do is either use the sh script in the integration-test folder run-local-test.sh (it's calling this config yaml file as `localconfig.yaml`, [see](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/run-local-tests.sh#L54)), or pass the following property when running maven command: `-Dtest.credentials.file=your-localconfig.yaml`